hazelcast/src/main/java/com/hazelcast/jet/impl/operation/InitExecutionOperation.java

```diff
@@ -104,7 +104,7 @@ public InitExecutionOperation(long jobId, long executionId, int coordinatorMembe
 
     @Override
     public ExceptionAction onInvocationException(Throwable throwable) {
-        return isRestartableException(throwable) ? THROW_EXCEPTION : super.onInvocationException(throwable);
+        return isTopologyException(throwable) ? THROW_EXCEPTION : super.onInvocationException(throwable);
```

Comment:
we can remove this `onInvocationException` override, it is exactly the same as `AsyncOperation`

Reply:
fixed